tests/conftest.py

```diff
@@ -12,6 +12,7 @@ def alert_dict():
         'area_names': [],
         'message_type': 'alert',
         'identifier': '1234',
+        'starts': datetime(2021, 4, 21, 11, 25, tzinfo=pytz.utc),
```

Comment:
This is a bit picky, but I think we've swapped the meaning of `starts/onset` and `sent` dates when compared to their use in CAP which might get a little confusing.

In CAP the sent date is when the alert is displayed, the onset/starts date is when the thing the alert pertains to begins. In this case I think the "sent" date would be used to determine visibility of the alert.

Reply:
Nice catch @rjbaker! I think it's important we're consistent with terminology, so we don't make mistakes. I've added a third commit to swap the meaning of the two fields, with a test to check they behave as expected.